code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -1333,6 +1333,12 @@
 						H.blood_volume += 10
 					for(var/datum/disease/critical/heart_failure/HF in H.viruses)
 						HF.cure() //Won't fix a stopped heart, but it will sure fix a critical one. Shock is not fixed as healing will fix it
+					var/list/organs_list = list()
+					organs_list += H.internal_organs
+					organs_list += H.bodyparts
+					for(var/obj/item/organ/O as anything in organs_list)
+						O.germ_level = 0
+					organs_list.Cut()
```

Comment:
```suggestion
					for(var/obj/item/organ/O as anything in (H.internal_organs + H.bodyparts))
						O.germ_level = 0
```
This should also do the trick. Be sure to test it just in case

Reply:
Tested it just in case, and it worked